src/signals/incident-management/containers/Dashboard/components/Filter/Filter.tsx

```diff
@@ -17,11 +20,24 @@ type Props = {
 
 export const Filter = ({ callback }: Props) => {
   const [filterActiveName, setFilterActiveName] = useState('')
+  const { dashboardFilter, setDashboardFilter } = useContext(
+    IncidentManagementContext
+  )
+
+  const location = useLocationReferrer() as { referrer: string }
 
   const methods = useForm<{ [key: string]: Option }>({
     defaultValues: Object.fromEntries(
       ['department', 'category', 'priority', 'punctuality', 'district'].map(
-        (name) => [name, { value: '', display: '' }]
+        (name) => [
+          name,
+          (location?.referrer === INCIDENTS_URL &&
```

Comment:
when the referrer is the incidents_url (also when going back through the history button (browser prev button), use the saved dashboard filter as default values.( asked ux )